src/mlpack/bindings/python/tests/test_python_binding.py

```diff
@@ -608,5 +648,142 @@ def testModelForceCopy(self):
+
+    """
+    Test that we pass wrong type and get back TypeError
+    """
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in=10,
+                                                              int_in=12,
+                                                              double_in=4.0,
+                                                              flag1=True))
+
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in='hello',
+                                                              int_in=10.0,
+                                                              double_in=4.0,
+                                                              flag1=True))
+
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in='hello',
+                                                              int_in=12,
+                                                              double_in='bad',
+                                                              flag1=True))
+
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in='hello',
+                                                              int_in=12,
+                                                              double_in=4.0,
+                                                              flag1=True,
+                                                              flag2=10))
+
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in='hello',
+                                                              int_in=12,
+                                                              double_in=4.0,
+                                                              flag1=True,
+                                                              matrix_in= 'bad'))
+
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in='hello',
+                                                              int_in=12,
+                                                              double_in=4.0,
+                                                              flag1=True,
+                                                              matrix_in= 1))
+
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in='hello',
+                                                              int_in=12,
+                                                              double_in=4.0,
+                                                              flag1=True,
+                                                              matrix_and_info_in = 10.0))
+
+    self.assertRaises(TypeError, lambda : test_python_binding(string_in='hello',
+                                                              int_in=12,
+                                                              double_in=4.0,
+                                                              flag1=True,
+                                                              copy_all_inputs = np.random.randn(10)))
```

Comment:
I think this line is too long (and maybe others too), can you make sure these are all 80 characters?  Or I can do it during merge, up to you.  Our style checker doesn't run on Python code. :)